documentation/ssl.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: documentation
+menu: ssl
+subtitle: SSL support
+---
+# SSL support
+
+In many scenarios, maintaining a secure connection to the database is important. Flyway can easily be configured to
```

Comment:
I think 'In many scenarios, maintaining a secure connection to the database is important' sort of goes without saying :)

Reply:
I'll reword it a bit, but it's always good to err on the side of caution around how much the readers actually know.